src/constants/index.ts

```diff
@@ -27,6 +29,26 @@ export const PASSWORD_ENTROPY_STEP = 6;
 
 export const DEFAULT_KYC_STEPS = ['email', 'phone', 'profile', 'document', 'address'];
 
+export const DEFAULT_MARKET_HEADERS = ['Pair', 'Price', '24h Change'];
+
+export const DEFAULT_ORDER_TYPES: DropdownElem[] = [
+    'Limit',
+    'Market',
+];
+export const AMOUNT_PERCENTAGE_ARRAY = [0.25, 0.5, 0.75, 1];
+
+export const DEFAULT_MARKET = {
+    id: '',
+    name: '',
+    base_unit: '',
+    quote_unit: '',
+    min_price: '',
+    max_price: 0,
+    min_amount: 0,
+    amount_precision: 0,
+    price_precision: 0,
+};
```

Comment:
For me it's a wrong place to put default market. Need to create additional file for all default objects

Reply:
I'm not going to change logic which was written previously. This PR is about hooks

Reply:
good idea, bug this PR is about hooks and I may be resolved on next PRs

Reply:
I suppose the idea of this PR not about "hooks", it's rather about refactoring of Markets. Hooks - just technology that allows us to optimize our code.

Reply:
This PR is to finish refactoring which @VladyslavP started and go to next stuff.